gatsby-config.js

```diff
@@ -1,6 +1,7 @@
 const siteUrl = `https://greenonsoftware.com`
 const siteName = "GreenOn Software"
 const siteDescription = `A place for people who love programming and personal development.`
+const discordUrl = "https://discord.gg/uJa3KUfFpj"
```

Comment:
We need to add at the end `/`. It's because of 302 redirection status for opening it first time. Crawlers in Ahrefs platform will throw warning if there is some url without / at the end.